unity-renderer/Assets/DCLServices/ApplicationFocus/ApplicationFocusService.cs

```diff
@@ -0,0 +1,43 @@
+using System;
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class ApplicationFocusService : IApplicationFocusService
+{
+
+    public event Action OnApplicationFocus;
+    public event Action OnApplicationFocusLost;
+
+    public void Initialize()
+    {
+        Application.focusChanged += FocusChange;
+    }
+    private void FocusChange(bool focus)
+    {
+        if (focus)
+        {
+            OnFocusGained();
+        }
+        else
+        {
+            OnFocusLost();
+        }
```

Comment:
Should we track current status to just call the events on changes?  I saw in your previous implementations that you were checking it to avoid calling `OnFocusGained` or `OnFocusLost` more than one time.

If that's an impossible situation you can ignore this :D

Reply:
I think its an impossible situation; but just in case I will add it.